receiver/kafkametricsreceiver/metadata.yaml

```diff
@@ -9,7 +9,16 @@ status:
   codeowners:
     active: [dmitryax]
 
+resource_attributes:
+  cluster_alias:
+    description: The alias name (string) of the cluster
+    enabled: true
```

Comment:
should this be enabled by default?

Reply:
Yes, We should disable it by default, I did the same, It will only be enabled when the new `cluster_alias` field is set in the config.